mlflow/gateway/handlers.py

```diff
@@ -0,0 +1,263 @@
+import pathlib
+from enum import Enum
+import json
+import os
+from pathlib import Path
+from pydantic import BaseModel, validator, parse_obj_as
+from pydantic.json import pydantic_encoder
+from typing import Optional, Union, List, Dict, Any
+import yaml
+
+from mlflow.exceptions import MlflowException
+from mlflow.gateway.constants import PROVIDERS
+from mlflow.gateway.utils import is_valid_endpoint_name, check_configuration_route_name_collisions
+from mlflow.protos.databricks_pb2 import INVALID_PARAMETER_VALUE
+
+
+class Provider(str, Enum):
+    UNSPECIFIED_PROVIDER = "custom"
```

Comment:
Reposting https://github.com/mlflow/mlflow/pull/8572/files#r1220544981

Reply:
sorry. missed this. Updated now.